### PR TITLE
Intial readme

### PR DESCRIPTION
**Pull request by ellisonbg**
Adding information about the summer team.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # jupyterlab-git
+
 A Git extension for JupyterLab
+
+This work is being done at Cal Poly during the summer of 2016.
```
Comment (ashutoshbondre) [inline README.md]: Change needed

Comment (nproctor) [inline README.md]: Should be 2017

Comment (hzarea) [inline README.md]: Change 2016 to 2017

Comment (zzhangjii) [inline README.md]: check if it works

**Pull request merged**